modules/sfm/src/libmv_light/libmv/simple_pipeline/bundle.cc

```diff
@@ -494,8 +499,13 @@ void EuclideanBundleCommonIntrinsics(
       constant_translation.push_back(4);
       constant_translation.push_back(5);
 
+#if CERES_VERSION_MAJOR >= 2 && CERES_VERSION_MINOR >= 1
+      constant_translation_parameterization =
```

Comment:
You mean `constant_translation_manifold`?

Reply:
indeed, thx. Fixed